providers/zfs-localpv-provisioner/run_litmus_test.yml

```diff
@@ -34,6 +34,13 @@ spec:
           - name: ZFS_DRIVER_IMAGE
             value: ''
 
+            ## This env will be used to select zfs-operator file for 
+            ## specific Operating system on which k8s cluster is created
+            ## For ubuntu based cluster give value as `ubuntu` (for both either 18.04 or 16.04)
+            ## For CentOS based cluster give value as either `centos7` or `centos8` accordingly
+          - name: OS_NAME  
+            value: ''  
```

Comment:
is this file different for centos7 and centos8?

Reply:
Yes, different for both